proto/rill/runtime/v1/resources.proto

```diff
@@ -121,6 +121,11 @@ message ModelSpec {
   google.protobuf.Struct stage_properties = 17;
   string output_connector = 1;
   google.protobuf.Struct output_properties = 12;
+  // retry is optional.
+  optional uint32 retry_attempts = 26;
+  optional uint32 retry_delay = 27;
```

Comment:
I suggest calling this `retry_delay_seconds` (not in the YAML, but in the internal proto)